tests/workers/test_glue_job_worker.py

```diff
@@ -0,0 +1,139 @@
+from unittest.mock import MagicMock
+from uuid import uuid4
+
+import boto3
+import pytest
+from moto import mock_glue
+from prefect.server.schemas.core import FlowRun
+
+from prefect_aws.workers.glue_job_worker import (
+    GlueJobWorker,
+    GlueJobWorkerConfiguration,
+    GlueJobWorkerResult,
+)
+
+
+@pytest.fixture(scope="function")
+def glue_job_client(aws_credentials):
+    with mock_glue():
+        yield boto3.client("glue", region_name="us-east-1")
+
+
+@pytest.fixture
+def flow_run():
+    return FlowRun(flow_id=uuid4(), deployment_id=uuid4())
+
+
+def test_get_client(aws_credentials):
+    with mock_glue():
+        job_worker_configuration = GlueJobWorkerConfiguration(
+            job_name="test_glue_job_name"
+        )
```

Comment:
I think if you provide an `AwsCredentials` object here with a region, it will resolve the test errors you're seeing. The credentials from `mock_aws_credentials` are only loaded in the tests if you specify one of the two profile names.